### ISSUE TITLE
A11y_SQLAzureDataStudio_NewDeployment_ SQLServerBigDataCluster_Screenreader: Voiceover doesn’t announce as ‘Required’ for ‘I accept…License Terms’ checkbox. 

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9289]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOS;#MAS1.3.1;#Screenreader;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3
Additional Details:
Mac OS:10.15.3
MAS References: MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options
5. Launch voiceover
6. Navigate to ‘I accept…License Terms’ checkbox using Control+ Option+ arrow key

**Actual:**
Voiceover doesn’t announce as ‘Required’ for ‘I accept…License Terms’ checkbox when navigated using Control+ Option + arrow key.

**Same issue is observed in Database Dashboard screen:** 1. Launch Azure Data Studio.2. Connect to server.3. In the Connections pane, navigate to server --> database --> right click on any database & select Manage.

**Expected:**
Voiceover should announce as ‘Required’ for ‘I accept…License Terms’ checkbox when navigated using Control+ Option + arrow key.

**User Impact:**
Screen reader users not get information that ‘I accept…License Terms’ checkbox is mandatory field.

**Attachment link for Reference:**
[9289_A11y_SQLAzureDataStudio_NewDeployment_ SQLServerBigDataCluster_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4274838/9289_A11y_SQLAzureDataStudio_NewDeployment_.SQLServerBigDataCluster_Screenreader.zip)

**Comment by alanrenmsft** [comment]
this has been fixed in the latest insider build

**Comment by sspatil123** [comment]
GithubTags:#Closed; Version 1.41.1
![image](https://user-images.githubusercontent.com/85863999/216963552-e3100d44-6784-4b47-a3f8-31541abac4a6.png)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
